tensorflow_datasets/core/download/download_manager.py

```diff
@@ -259,7 +263,7 @@ def callback(val):
       checksum, dl_size = val
       return self._handle_download_result(
           resource, download_dir_path, checksum, dl_size)
-    return self._downloader.download(url, download_dir_path).then(callback)
+    return self._downloader.download("%s%s" % (url, download_type), download_dir_path).then(callback)
```

Comment:
I'm confused that you first split the url line 244 and then merge it back here. Why split it it in the first place ?